maker-core/cvc.md

```diff
@@ -1,41 +1,42 @@
 
-Each CVC produces Governance Strategy Position documents and Scope Framework Position Documents to update the [Scope Frameworks](list-of-scopes.md). This set of documents provides general guidelines for how [Constitutional Delegates (CDs)](delegates.md) that follow their governance strategy should vote.
+The documents produced by the AVCs abstract away the minutiae of day-to-day governance. Since Aligned Delegates execute AVC strategies, they allow MKR Holders to be reasonably informed and secure with regard to the general direction of the chosen delegate.
 
-Position Documents abstract away the minutiae of day-to-day governance. Since Constitutional Delegates execute CVC strategies, they allow MKR Holders to be reasonably informed and secure with regard to the general direction of the chosen delegate.
+According to their own strategy, AVCs fall somewhere in the spectrum that goes from **Hawk** to **Dove**:
+  - Hawkish AVCs focus more on short-term income and asset accumulation, tending to prefer higher spreads and lower budgets
+  - Dovish AVCs focus more on long-term growth and innovation, tending to prefer lower spreads and higher budgets
+  - Balanced AVCs try to find a middle ground between the two
+
+The Hawk, Dove, Balanced is a continuous spectrum, so many different hawkish or dovish AVCs can exist that are more or less Balanced.
 
-In order for a Voter Committee to be considered a CVC, it must have at least one member who is a [Constitutional Conserver (CC)](constitutional-conservers.md) and a verified MKR holder. CVCs are also subject to additional eligibility requirements, detailed below.
 
 ## Eligibility Requirements
+In order for a voter committee to be considered an AVC, it must have at least one member who is an [Alignment Conserver (AC)](constitutional-conservers.md) and a verified MKR holder.
 
-Voter Committees must meet certain requirements for at least a quarterly governance cycle to become CVCs:
-- At least one registered member must be a CC and a verified MKR holder
-- For each quarterly governance cycle, CVCs must:
-   1. Create a Governance Strategy Position and 12 Scope Framework Position documents
-   2. Hold one CVC Subcommittee meeting for each scope
+To become Active AVCs, AVCs are subject to additional eligibility requirements. For each quarterly governance cycle they must:
+   - Create one Aligned Governance Strategy document and five Aligned Scope Proposals
+   - Hold two AVC Subcommittee meetings for each scope
 
-The [Arbitration Scope](https://mips.makerdao.com/mips/details/MIP113) recognizes eligible CVCs. It can revoke this recognition anytime the CVC fails to meet all eligibility requirements.
+The [Governance Scope](https://mips.makerdao.com/mips/details/MIP113) recognizes Active AVCs. It can revoke this recognition anytime the AVC fails to meet all eligibility requirements.
```

Comment:
(Not sure what "recognizes" means here)

Reply:
https://mips.makerdao.com/mips/details/MIP101#2-5-3-aligned-voter-committee-recognition-process